other/java/client/src/main/java/seaweedfs/client/SeaweedWrite.java

```diff
@@ -123,13 +125,20 @@ private static String multipartUpload(String targetUrl,
                                           final byte[] bytes,
                                           final long bytesOffset, final long bytesLength,
                                           byte[] cipherKey) throws IOException {
+        MessageDigest md = null;
+        try {
+            md = MessageDigest.getInstance("MD5");
+        } catch (java.security.NoSuchAlgorithmException e) {
```

Comment:
should fail this or check whether md is null when using it later.

Reply:
Letting this fail would change the interface of the API function (`throws IOException` -> `throws IOException, NoSuchAlgorithmException`)

`md` will never be `null`, "MD5" is supported on all JVM